bin/.gitignore

```diff
@@ -1 +0,0 @@
-/com/
```

Comment:
Move `.gitignore` to the root folder. add bin/ to the list and delete the bin folder from your local. Push the changes without rebuilding so that the repo sees that it should delete the files. The next time you rebuild after committing, those bin files will be ignored.